docs/auth/README.md

```diff
@@ -0,0 +1,128 @@
+# Running Lighthouse on Authenticated Pages
+
+Default runs of Lighthouse load a page as a "new user", with no previous session or storage data. This means that pages requiring authenticated access do not work. There are multiple ways to run Lighthouse on an authenticated page - this document focuses on the most flexible approach ([Puppeteer](https://pptr.dev)), but mentions other approaches briefly at the end.
+
+If you just want to view the code, see [./example-lh-auth.js](example-lh-auth.js).
+
+## The Example Site
+
+There are two pages on the site:
+
+1. `/` - the homepage
+2. `/dashboard`
+
+The homepage shows the login form, but only to users that are not signed in.
+
+The dashboard shows a secret to users that are logged in, but shows an error to users that are not.
+
+The server responds with different HTML for each of these pages and session states, so there are four different pages that must have passable Lighthouse SEO scores.
+
+You can run this server locally if you like:
+
+```sh
+# be in root lighthouse directory
+yarn # install global project deps
+cd docs/auth
+yarn # install deps related to just this documentation
+yarn start # start the server on http://localhost:8000
+```
+
+## Puppeteer
+
+We can use Puppeteer - a browser automation tool - to manipulate a setup a session programatically.
+
+1. Launch a new browser.
+1. Navigate to the login page.
+1. Fill and submit the login form.
+1. Run Lighthouse using the same browser.
+
+First we launch Chrome:
```

Comment:
I usually avoid "we" because it makes the subject of the action ambiguous. In this case "we're" not doing anything, we're telling the reader to do something. In this context it's more of a nitpick --- I don't think anyone will get lost in this doc --- but sometimes it does become an instructional issue.

https://developers.google.com/style/person